content/blog/2019/06/2019-06-10-use-casc-for-exws.adoc

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "Use Terraform with External Workspace Manager to Support EFS"
+tags:
+- jenkins
+- gsoc
+author: yufei_zhang
+description: "A brief example of using Terraform with EFS" # optional
+opengraph:
+---
+
+=== Why you want to use External Workspace Manager with EFS
+What is Terraform : A CasC software, like CloudFormation and Ansible designed for managing cloud resources. It can free you from managing cloud resource by console, and can be version controlled using git as it is configuration-as-code. Currently it supports AWS and Azure best. AWS CloudFormation is for AWS only.
+What is extenal-workspace-manager : It is a pipeline plugin let you custom your workspace path, you can mount shared disk to multiple nodes to share workspaces.
```

Comment:
English: It is a pipeline plugin which customizes the workspace path. The same disk can be mounted to multiple nodes to share resources.